README.md

```diff
@@ -278,36 +222,87 @@ Check out `tune --help` for all possible CLI commands and options. For more info
 
 &nbsp;
 
-## Design Principles
+---
+
+## Llama3 and Llama3.1
+
+torchtune supports fine-tuning for the Llama3/Llama3.1 8B, 70B, and 405B size models. You can fine-tune the 8B model with LoRA, QLoRA and full fine-tunes on one or more GPUs. You can also fine-tune the 70B model with QLoRA on a single device or LoRA and full-finetunes on multiple devices. Finally, you can fine-tune the 405B model on a single node with QLoRA. For all the details, take a look at our [tutorial](https://pytorch.org/torchtune/main/tutorials/llama3.html).
```

Comment:
imo, less is more. No need to specify 8b/70b/405b in 3 different lines. Something like this should be good

"torchtune supports fine-tuning for the Llama3/Llama3.1 8B, 70B, and 405B size models with LoRA, QLoRA and full fine-tunes on one or more GPUs, depending on the model size. For all the details, take a look at our [tutorial](https://pytorch.org/torchtune/main/tutorials/llama3.html)."